.rubocop_todo.yml

```diff
@@ -1,3 +1,9 @@
 Metrics/ParameterLists:
   Exclude:
     - 'lib/rights_database/rights.rb'
+Metrics/AbcSize:
```

Comment:
Can we remove rubocop and add standardrb?

Reply:
https://github.com/hathitrust/hathi_search_client/pull/17
Not going to fold into this PR because it makes a mess.